### DISCUSSION TITLE
One hand carry everything.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#84] [Ideas]
Of course idea taken from another project and again only as a suggestion, perfectly understandable if it not feasible or not needed. if accepted hopefully it is also togge-able, Thank you so much for your hardwork and dedication to this project!

**Comment by winstxnhdw** [comment]
Sure, I'll add it soon.

**Reply by yujispin990** [reply]
you're the best @winstxnhdw ❤️

**Reply by winstxnhdw** [reply]
Done. I didn't make it toggleable. But it should not be animated to others and yourself as a one-handed item, but you'll gain the benefits of pocketing them, which is what I assume you want it for. Hope you can test it for me.
